src/java/org/apache/cassandra/audit/AuditLogManager.java

```diff
@@ -79,7 +77,7 @@ private AuditLogManager()
         }
         else
         {
-            logger.debug("Audit logging is disabled.");
+            logger.info("Audit logging is disabled.");
```

Comment:
Why the change?

Reply:
From an auditing point of view I think it's important to know whether audit logging is enabled or disabled when you start the instance.